quickstarts/apdex-optimizer/config.yml

```diff
@@ -0,0 +1,78 @@
+# Sets the URL name of the quickstart on public I/O (required)
+slug: apdex-optimizer
+
+# Displayed in the UI (required)
+title: Apdex Optimizer
+
+# Long-form description of the quickstart (required)
+description: |
+  # Summary of Apdex Optimizer Dashboard
+
+## 1. Apdex Score Overview:
+- Timeseries chart displaying Apdex scores for each APM application.
+- Highlights performance trends over time.
+
+## 2. Apdex Breakdown:
+- Detailed breakdown table showcasing Apdex scores for different aspects of your application.
+- Identifies areas of improvement and potential bottlenecks.
+
+## 3. Transaction-level Insights:
+- In-depth Apdex information on individual transactions.
+- Pinpoints specific transaction performance for focused optimization.
+
+## 4. Threshold Recommendations:
+- Application Apdex T Value Recommendation table guides optimal threshold settings.
+- Helps fine-tune Apdex parameters for enhanced performance monitoring.
+
+## 5. Transaction Apdex Values:
+- Transaction Apdex Values table provides granular insights into transaction-level performance.
+- Enables precise adjustments for optimal user satisfaction.
+
+### Actionable Takeaways:
+- Identify and address performance trends through Apdex score patterns.
+- Optimize specific transactions based on detailed Apdex insights.
+- Fine-tune Apdex thresholds for applications to improve overall monitoring effectiveness.
+
+### Benefits:
+- Enhanced visibility into application performance.
+- Proactive identification of potential issues.
+- Improved user satisfaction through targeted optimizations.
+
+This comprehensive Apdex Optimizer dashboard empowers you with actionable data to make informed decisions, ensuring your applications perform at their best.
```

Comment:
I think we need these lines to be indented passed the `description` key. At the moment we would just get the description to be `# summary of apdex optimizer dashboard`.

If we indent these lines by 1, it should fix it!

Reply:
done

Reply:
Hey @DarrenDoyle thanks for making that change. Unfortunately, you increased the indent by `2` rather than by `1`. I think this might cause the text to render incorrectly in the product. Can you please update this?

Thanks.